specification/storage/Microsoft.BlobStorage/routes.tsp

```diff
@@ -1715,7 +1715,6 @@ namespace Storage.Blob {
             ...ContentMd5ResponseHeader;
             ...ContentCrc64ResponseHeader;
             ...BlobSequenceNumberResponseHeader;
-            ...DateResponseHeader;
```

Comment:
@vincenttran-msft why did you remove the date headers in this comment? Shouldnt you be using the private version of the headers?

Reply:
Good catch, I thought I had fixed this during the in-meeting review but must have been missed.